LayoutTests/imported/w3c/web-platform-tests/css/css-box/margin-trim/block-container-float-multiple-formatting-contexts-inline-layout.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="author" title="Sammy Gill" href="mailto:sammy.gill@apple.com">
+<link rel="help" href="https://drafts.csswg.org/css-box-4/#margin-trim-block">
+<style>
+container {
+    margin-trim: block;
+    display: flow-root;
+    width: max-content;
+}
+item {
+    display: block;
+    width: 25px;
+    height: 100px;
+    margin-block-end: 50px;
+    float: left;
+}
+span {
+    display: inline-block;
+    width: 25px;
+}
```

Comment:
Same as above, not sure why <span> is turned into inline-block.

Reply:
Should be changed now